### ISSUE TITLE
Long build times

### ISSUE THREAD

**Issue by neoromantic** [nathantreid/meteor-css-modules#91]
So, I have quite small (yet) project, built with latest Meteor, meteor-css-modules and react-toolbox.

Currently, build (meteor build) time on my latest MacBook pro is ~5 minutes, half of which is taken by nathantreid:css-modules.

Is there any known way to improve build speed?

**Comment by neoromantic** [comment]
```
   START 1 building for web.browser          /
     START 2 linking the program             /
     DONE 2 linking the program took 12      /
     START 2 loading plugin `compileTemplatesBatch` from package `templating-compiler`
     DONE 2 loading plugin `compileTemplatesBatch` from package `templating-compiler` took 894
     START 2 running registerCompiler callback in package templating-compiler
     DONE 2 running registerCompiler callback in package templating-compiler took 1
     START 2 loading plugin `compileCoffeescript` from package `coffeescript`
     DONE 2 loading plugin `compileCoffeescript` from package `coffeescript` took 395
     START 2 running registerCompiler callback in package coffeescript
     DONE 2 running registerCompiler callback in package coffeescript took 1
     START 2 processing files with ecmascript (for target web.browser)
     DONE 2 processing files with ecmascript (for target web.browser) took 20551
     START 2 processing files with templating-compiler (for target web.browser)
     DONE 2 processing files with templating-compiler (for target web.browser) took 700
     START 2 processing files with meteor (for target web.browser)
     DONE 2 processing files with meteor (for target web.browser) took 2
     START 2 processing files with coffeescript (for target web.browser)
     DONE 2 processing files with coffeescript (for target web.browser) took 913
     START 2 processing files with static-html (for target web.browser)
     DONE 2 processing files with static-html (for target web.browser) took 11
     START 2 processing files with nathantreid:css-modules (for target web.browser)
     DONE 2 processing files with nathantreid:css-modules (for target web.browser) took 181533
     START 2 processing files with universe:i18n (for target web.browser)
     DONE 2 processing files with universe:i18n (for target web.browser) took 17
```

Here's what I get from build profiler. As you can see, nathantreid:css-modules takes >50% of build time, more than 3 minutes.

All I have in my project is maybe 10 css files + react-toolbox.

node_modules folder is excluded like this:

```
    "ignorePaths": [
      "node_modules/(?!react-toolbox/lib)"
    ],
```

Any guidance?

**Comment by akanix42** [comment]
That's insanely long! I haven't seen any build times like that yet but I'm aware of some others experiencing that issue with node_modules. What's your build time for https://github.com/neoromantic/meteor-cssmodules-reacttoolbox?

**Comment by neoromantic** [comment]
Overall build time is about 40-60 seconds, nathantreid:css-modules takes only 3 seconds of that time.

**Comment by akanix42** [comment]
Can you please set 
```json
"cssModules": {
  "enableProfiling": true
}
```
in your `package.json` and report the console output when you run your project? That should help to narrow down where the bottleneck is occurring.

Thanks!

**Comment by neoromantic** [comment]
I'm sorry for long answer, got caught up in work.

Anyway, I can't put my finger on what's happened, but currently my build time is adequate. Dunno what changed exactly, will monitor this issue for some time.

**Comment by akanix42** [comment]
I'm going to close this for now since it's been idle for a while. I suspect this was fixed in [v2.5.4](https://github.com/nathantreid/meteor-css-modules/blob/master/CHANGELOG.md#user-content-254---2017-05-17).
Feel free to reopen it if you continue to run into performance issues.
